### PR TITLE
POL-260 Google unattached volumes unhandled errors when executing

### PR DESCRIPTION
**Pull request by SheikSena**
## Issues Resolved

- If APIs and Services are not enabled for a project.
- If the project is shutdown.  

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

Annotation 2020-04-07 161552

### CODE REVIEW

File: cost/google/unattached_volumes/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.3
+
+- Bug fixes on unhadled errors when executing
```
Comment (cdwilhelm) [inline cost/google/unattached_volumes/CHANGELOG.md:5]: check your spelling

File: cost/google/unattached_volumes/README.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 This Policy Template scans all volumes in the given account and identifies any unattached volumes that have been unattached for at least the number of user-specified days. If any are found, an incident report will show the volumes, and related information and an email will be sent to the user-specified email address.
 
+If APIs & services are not enabled for a project, the policy will skip that particular project and executes it. On the next run if services are enabled, then the project will be taken into consideration.
```
Comment (cdwilhelm) [inline cost/google/unattached_volumes/README.md:7]: I don't understand the part **and execute it**.  It executes what.  I thought it is skipped.  Can you clarify and make this more clear.  Maybe I'm not understanding.  thx

File: cost/google/unattached_volumes/CHANGELOG.md (hunk 1 above)
Comment (SheikSena) [reply cost/google/unattached_volumes/CHANGELOG.md:5]: Done !

File: cost/google/unattached_volumes/README.md (hunk 2 above)
Comment (SheikSena) [reply cost/google/unattached_volumes/README.md:7]: Fixed It. Please let me know your thoughts.

**Pull request merged**